plugin/entryBase.go

```diff
@@ -67,9 +69,19 @@ func (e *EntryBase) ID() string {
 	return e.id
 }
 
-// Attr returns the entry's attributes
+// Attr returns the entry's attributes. The default return value
+// is a zero'ed attributes struct. If e.Ctime is set, then
+// the returned attributes struct's Ctime, Mtime, and Atime fields
+// are set to this value.
+//
+// Entries can override Attr if they'd like to return a different
+// set of attributes.
 func (e *EntryBase) Attr(ctx context.Context) (Attributes, error) {
-	return e.attr, nil
+	return Attributes{
+		Ctime: e.Ctime,
+		Mtime: e.Ctime,
+		Atime: e.Ctime,
```

Comment:
Should plugin authors worry about `Atime` ?  It seems like if you do anything with any entry in Wash, you're "accessing" it so `Atime` should always be `time.Now()`. Hm, maybe for sshfs-type files it could be useful.

Reply:
`Atime` generally means when it was "read". For actual files it makes sense. For other things I'm not sure.

Reply:
Oh ok. Might as well keep it then.